models/mpas_atm/update_mpas_states.f90

```diff
@@ -14,83 +14,73 @@ program update_mpas_states
 !         The update_mpas_states_nml namelist defines the input and output file
 !         name lists for all ensemble members.
 !         The input list should be matched with output_state_file_list in &filter_nml.
-!         
+!
+!  variables that are not wind, copied from one file to the another
+!  variables that are wind, reconstructed
```

Comment:
good set of comments but I think the comma is confusing 

"
!  variables that are not wind are copied from one file to the another
!  variables that are wind are reconstructed
"

Reply:
fixed in 52b87fcae0ef39b320f699c003c66b7f74f6d86